Modix/Modules/ModerationModule.cs

```diff
@@ -18,39 +18,53 @@ public ModerationModule(IModerationService moderationService)
             ModerationService = moderationService;
         }
 
+        public async Task AddConfirmationEmoji()
```

Comment:
I would just make this `AddConfirmation` or `ReplyConfirmation` as it's not NECESSARILY gonna be an Emoji. ;)